themes/digital.gov/src/scss/new/guides/_guide-content.scss

```diff
@@ -0,0 +1,37 @@
+@use "uswds-core" as *;
+
+.dg-guide__content-body {
+  margin-bottom: units(5);
+  overflow: hidden;
+
+  p {
+    max-width: none;
+  }
+}
+
+.dg-guide__content-header {
+  margin: units(5) 0;
+
+  &-deck {
+    @include u-font("sans", "xl");
+    color: color("primary-darker");
+    font-weight: font-weight("light");
+    margin-top: 0;
+  }
+
+  &-title {
+    @include u-font("sans", "2xl");
+    font-weight: font-weight("bold");
+    margin-bottom: 0;
+  }
+}
+
+.dg-guide__nav-container {
+  margin: units(5) 0;
+  position: sticky;
+  top: units(15);
+}
+
+html {
+  scroll-padding-top: units(15);
+}
```

Comment:
Can you add a comment as to what this does/why it's necessary?

---

**TODO:** Let's follow up on this with a card in backlog to create a partial for global styles. That way we can easily update in the future.

Reply:
Card created [here](https://trello.com/c/u2q0NNah/2397-dg-create-global-styles-partial).